web/film-club-client/src/App.js

```diff
@@ -73,14 +73,16 @@ function App() {
 
   // Get first batch of film data
   const getUpcomingMovies = async () => {
-    const response = await fetch("/films/upcoming")
+    const page = (movies.length / 20) + 1
+    console.log(`getUpcomingMovies called for page: ${page}`)
+    const response = await fetch(`/films/upcoming?page=${page}`)
     const data = await response.json()
-    setMovies(data.films)
+    setMovies(movies.concat(data.films))
   }
 
-  useEffect(() => {
-    getUpcomingMovies()
-    }, [])
+  // useEffect(() => {
+  //   getUpcomingMovies()
+  //   }, [])
```

Comment:
💅  Feel free to delete commented-out code! If you _really_ need it, it's in your commit history!